package.json

```diff
@@ -164,11 +164,18 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after Build",
+      "description": "Run clean command after every successful build.",
```

Comment:
How is this wording?
> Run clean command after every successful build. **Note:** Default clean patterns includes files that you might want to keep, e.g. PDF, DVI and PS. Make sure to have a look at the default patterns before turning this setting on.

I can't do a new line :/ and the bold is also not much more readable.

<img width="769" alt="screen shot 2017-03-01 at 18 17 25" src="https://cloud.githubusercontent.com/assets/8694384/23472793/4c683864-feae-11e6-9877-6aa912640bbf.png">